app/app/controllers/cbv/base_controller.rb

```diff
@@ -11,6 +13,7 @@ def set_cbv_flow
       end
 
       @cbv_flow = invitation.cbv_flow || CbvFlow.create_from_invitation(invitation)
+      NewRelicEventTracker.track('ClickedCBVInvitationLink', { timestamp: Time.now.to_i })
```

Comment:
This is a good start. I'd imagine that we'd want to collect things such as the `session_id` and `invitation_id` as well. 

```suggestion
NewRelicEventTracker.track('ClickedCBVInvitationLink', { timestamp: Time.now.to_i, session_id: session_id, invitation_id: invitation.id })
```
Something like that.